### DISCUSSION TITLE
After retargeting with Metahuman, there is a hang-out in version 5.4, but there was no such problem in version 5.3.

### DISCUSSION THREAD

**Discussion by mustafakalaycii** [Sixze/ALS-Refactored#546] [Q&A]
After you released version 5.4 of the project, I retarget as in this video : https://www.youtube.com/watch?v=RT8TqGrN13g&t=97s 
But as you can see in the video, there was a hanging problem
And this problem does not occur when the project version is 5.3, but unfortunately I need version 5.4
Can you help me how to fix this problem? If there is any other Metahuman retarget method that you know, you can share it with me, I would be very pleased
Note: There is no such problem in the project itself, only when I make a metahuman retarget, I encounter such a hang-up in the project I am using version 5.4

https://github.com/user-attachments/assets/afd613c2-ec45-44ec-ba38-65ed92d8e823

https://github.com/user-attachments/assets/543bd7cf-e19d-4001-996e-493cededa337

https://github.com/user-attachments/assets/fa17432a-98d6-45e7-a27a-f79b8b406123

**Comment by ameaninglessname** [comment]
You could use rewind debugger to find out what happens exactly in those wired frames.

**Reply by mustafakalaycii** [reply]
I'll do as you say and show it in the form of a video @ameaninglessname
